src/components/side-panel/source-data-catalog.js

```diff
@@ -60,10 +60,17 @@ const DatasetTagWrapper = styled.div`
   color: ${props => props.theme.textColor};
   font-size: 11px;
   letter-spacing: 0.2px;
-  
+
   .dataset-color {
     margin-top: 5px;
   }
+
+  .dataset-name {
+    width: 11rem;
```

Comment:
Any way to use a constant or some variable so that this adapts to changes on the width of the side bar (if it gets updated)?

Reply:
Also, should this be `max-width` to retain the current behavior of showing the arrow at the end when the string is shorter than the width?